core-default/src/main/java/me/aurium/beetle/defaults/task/sync/TimerSleepBehavior.java

```diff
@@ -0,0 +1,18 @@
+package me.aurium.beetle.defaults.task.sync;
+
+import me.aurium.beetle.api.task.sync.BlockingBehavior;
+
+import java.util.concurrent.locks.LockSupport;
+
+public class TimerSleepBehavior implements BlockingBehavior {
+
+    //this needs review. I'm hoping that parknanos will halt the thread of it's invoker, which means behavior, which
```

Comment:
That's what it does, and that's also how code generally works. If a method blocks, it blocks the calling thread